plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -38,6 +38,12 @@ const FETCH_MORE_MESSAGES_THROTTLE_MS = isTesting() ? 0 : 500;
 const PAST = "past";
 const FUTURE = "future";
 
+const MENTION_RESULT = {
+  unreachable: 0,
+  over_members_limit: 1,
+  ignored: 2,
```

Comment:
What is ignored? I have read the ruby code but it's still unclear int what case it happens?

Reply:
This might be an overkill, but it's to cache results that don't match with any visible group.

Reply:
ok I see, could it be -1 ? maybe we go with invalid?

Reply:
so -1 id probably better in this case then, it leaves 2 free is we ever need if for something